packages/application-components/src/components/dialogs/internals/dialog-footer.tsx

```diff
@@ -49,19 +50,40 @@ const DialogFooter = (props: Props) => {
         margin-top: ${uiKitDesignTokens.spacing50};
       `}
     >
-      <Spacings.Inline scale="m" alignItems="center" justifyContent="flex-end">
-        <SecondaryButton
-          label={getFormattedLabel(props.labelSecondary, intl)}
-          onClick={props.onCancel}
-          iconLeft={props.iconLeftSecondaryButton}
-          {...filterDataAttributes(props.dataAttributesSecondaryButton)}
-        />
-        <PrimaryButton
-          label={getFormattedLabel(props.labelPrimary, intl)}
-          onClick={props.onConfirm}
-          isDisabled={props.isPrimaryButtonDisabled}
-          {...filterDataAttributes(props.dataAttributesPrimaryButton)}
-        />
+      <Spacings.Inline alignItems="center" justifyContent="space-between">
+        {props.leftAlignedFooterContent && (
+          <Spacings.Inline
+            scale="m"
+            alignItems="center"
+            justifyContent="flex-start"
+          >
+            {props.leftAlignedFooterContent}
+          </Spacings.Inline>
+        )}
+        <div
+          css={css`
+            margin-left: auto;
+          `}
+        >
```

Comment:
I suggest to remove this custom CSS and render an empty `<div />` in the place of the footer content to allow flexbox to do its work (meaning that there are always 2 child elements).

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3587/commits/e5dce4a0f9d506afde4f37109b791f1fb90c88d9